tox.ini

```diff
@@ -57,6 +62,8 @@ commands =
     docs: sphinx-build -b html -d {envtmpdir}/doctrees . ../bin/docs/html {posargs}
     pkg: python setup.py check -m -r -s
     pkg: check-manifest --ignore tox.ini,tests*,.github,.github/*,CONTRIBUTING.rst,docs,docs/*
+    system_tests: python tools/install_local_wheel.py --driver nidcpower
+    system_tests: coverage run --rcfile=tools/coveragerc_system_test --source nidcpower -m py.test src/nidcpower/system_tests --junitxml=junit-nidcpower-{envname}.xml {posargs}
     system_tests: python tools/install_local_wheel.py --driver nidmm
     system_tests: coverage run --rcfile=tools/coveragerc_system_test --source nidmm -m py.test src/nidmm/system_tests --junitxml=junit-dmm-{envname}.xml {posargs}
```

Comment:
You need to add the --append parameter here since it is no longer the first thing that runs.

Reply:
good catch.